src/backend/web/routes/index.js

```diff
@@ -45,13 +45,13 @@ if (process.env.NODE_ENV === 'development') {
     // Allow proxying the Gatsby dev server through our backend if PROXY_FRONTEND=1 is set in env
     router.use('/', createProxyMiddleware({ target: 'http://localhost:8000', changeOrigin: true }));
     // Or serve the static files in the Gatsby build directory
-  } else if (process.env.FRONTEND.toLowerCase() === 'gatsby') {
+  } else if (process.env.FRONTEND.toLowerCase() === 'gatsby' || process.env.FRONTEND === '') {
```

Comment:
Question: will `process.env.FRONTEND.toLowerCase()` break if `FRONTEND` isn't defined?  That is, should this be:

```
else if(!process.env.FRONTEND || process.env.FRONTEND === '' || process.env.FRONTEND.toLowerCase() === 'gatsby') {...}
```

Reply:
Oh, that's a good point, I'll make that change real quick